File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
  let count = 0;
  let max = 0;
  for (let i = 0; i < sentences.length; i++) {
    let word = sentences[i].split(" ");
    count = word.length;
    if (count > max) max = count;
  }
  return max;
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max_count = 0
        for sentence in range(len(sentences)):
            current_sentence = sentences[sentence].split()
            if len(current_sentence) > max_count:
                max_count = len(current_sentence)
        return max_count